src/modules/Jobs/modules/OrganizationJobListing/useOrganizationJobListing.tsx

```diff
@@ -47,13 +49,23 @@ export const useOrganizationJobListing = () => {
   const filterButtons: ButtonGroupItem[] = [
     {
       label: 'View all',
+      value: 'all',
       handleClick: () => {
         setFilter('all');
         setPage(1);
       },
     },
+    {
+      label: 'Active',
+      value: 'active',
+      handleClick: () => {
```

Comment:
`handleClick` is duplicated everywhere. please create a general function to accept only `name` for the `all|active|archived` filter